tasks/hello-world/kes_dist

```diff
@@ -0,0 +1 @@
+dist
```

Comment:
This is a symlink that allows us to have kes create duplicate zipfiles.   Kes filteres out lambdas with the same source, and our CMA override doesn't re-duplicate if it's the same lambda with different options. 

I could *fix* kes, but since we're moving from it, that seems unwarranted here.